frontend/src/pages/OverviewStandardsPage.vue

```diff
@@ -0,0 +1,266 @@
+<template>
+  <q-page class="column items-center">
+    <PageTitle title="Data Standards" subtitle="EMBER interoperability & reproducibility" />
+
+    <div class="q-mx-xl full-width max-w-screen-lg">
+      <!-- Overview -->
+      <q-card flat class="q-mt-lg">
+        <q-card-section>
+          <div class="text-h2 scrollable">Overview</div>
+          <p>
+            EMBER follows widely adopted, community-driven data standards to promote interoperability and reproducibility across multimodal neuroscience datasets. 
+            The framework currently supports 
+            <a href="https://bids.neuroimaging.io/index.html" target="_blank" rel="noopener" class="link">BIDS</a> 
+            and 
+            <a href="https://nwb.org/about-nwb/" target="_blank" rel="noopener" class="link">NWB</a> 
+            formats, along with hybrid layouts that combine both where appropriate, and anticipate accepting new standards. 
```

Comment:
```suggestion
            formats, along with hybrid layouts that combine both where appropriate. 
```

I'd suggest not planting abstract seeds of 'expectations' since 'who knows what will happen' in the future

Can (and should) always come back and modify the line if something changes later down the road